### PR TITLE
refactor(AM-ChangeAveshaRef): replaced avesha* labels to kubeslice.io

### PR DESCRIPTION
**Pull request by gourishkb**
Signed-off-by: gourishkb <104021126+gourishkb@users.noreply.github.com>

### CODE REVIEW

**Comment by gourishkb** [comment]
dev-charts PR https://github.com/kubeslice/dev-charts/pull/4

**Comment by gourishkb** [comment]
iperf test output 

```
------------------------------------------------------------
Client connecting to 192.168.1.1, TCP port 5201
TCP window size: 2.50 MByte (default)
------------------------------------------------------------
[  1] local 192.168.1.1 port 48488 connected with 192.168.1.1 port 5201
[ ID] Interval       Transfer     Bandwidth
[  1] 0.00-60.00 sec  37.8 MBytes  5.28 Mbits/sec

```

**Comment by rahulsawra98** [comment]
@gourishkb can you please run make test and paste the output here?

File: controllers/slicegateway/slicegateway.go [hunk 1]
```diff
@@ -331,10 +331,10 @@ func (r *SliceGwReconciler) deploymentForGatewayClient(g *kubeslicev1beta1.Slice
 	var vpnSecretDefaultMode int32 = 0644
 
 	certFileName := "openvpn_client.ovpn"
-	sidecarImg := "nexus.dev.aveshalabs.io/kubeslice/gw-sidecar:1.0.0"
+	sidecarImg := "nexus.dev.kubeslicelabs.io/kubeslice/gw-sidecar:1.0.0"
```
Comment (rahulsawra98) [inline controllers/slicegateway/slicegateway.go:334]: I think no such repo + image exists
nexus.dev.aveshalabs.io is a nexus repo name

File: controllers/slicegateway/slicegateway.go [hunk 2]
```diff
@@ -331,10 +331,10 @@ func (r *SliceGwReconciler) deploymentForGatewayClient(g *kubeslicev1beta1.Slice
...
 	sidecarPullPolicy := corev1.PullAlways
 
-	vpnImg := "nexus.dev.aveshalabs.io/avesha/openvpn-client.alpine.amd64:1.0.0"
+	vpnImg := "nexus.dev.kubeslicelabs.io/kubeslice/openvpn-client.alpine.amd64:1.0.0"
```
Comment (rahulsawra98) [inline controllers/slicegateway/slicegateway.go:337]: same as above

File: controllers/slicegateway/slicegateway.go (hunk 1 above)
Comment (gourishkb) [reply controllers/slicegateway/slicegateway.go:334]: Good catch! Reverting this

**Comment by gourishkb** [comment]
```
make test
/home/lenovo/code/namechange/worker-operator/bin/controller-gen rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
/home/lenovo/code/namechange/worker-operator/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
go fmt ./...
go vet ./...
KUBEBUILDER_ASSETS="/home/lenovo/.local/share/kubebuilder-envtest/k8s/1.22.1-linux-amd64" go test ./... -coverprofile cover.out
?   	github.com/kubeslice/worker-operator	[no test files]
?   	github.com/kubeslice/worker-operator/api/v1beta1	[no test files]
?   	github.com/kubeslice/worker-operator/controllers	[no test files]
?   	github.com/kubeslice/worker-operator/controllers/serviceexport	[no test files]
?   	github.com/kubeslice/worker-operator/controllers/serviceimport	[no test files]
?   	github.com/kubeslice/worker-operator/controllers/slice	[no test files]
?   	github.com/kubeslice/worker-operator/controllers/slicegateway	[no test files]
ok  	github.com/kubeslice/worker-operator/internal/cluster	20.057s	coverage: 43.2% of statements
?   	github.com/kubeslice/worker-operator/internal/gwsidecar	[no test files]
ok  	github.com/kubeslice/worker-operator/internal/hub/controllers	0.050s	coverage: 20.1% of statements
?   	github.com/kubeslice/worker-operator/internal/hub/hubclient	[no test files]
?   	github.com/kubeslice/worker-operator/internal/hub/manager	[no test files]
?   	github.com/kubeslice/worker-operator/internal/logger	[no test files]
ok  	github.com/kubeslice/worker-operator/internal/manifest	0.020s	coverage: 10.5% of statements
?   	github.com/kubeslice/worker-operator/internal/namespace/controllers	[no test files]
?   	github.com/kubeslice/worker-operator/internal/netop	[no test files]
?   	github.com/kubeslice/worker-operator/internal/networkpolicy	[no test files]
?   	github.com/kubeslice/worker-operator/internal/router	[no test files]
?   	github.com/kubeslice/worker-operator/internal/utils	[no test files]
ok  	github.com/kubeslice/worker-operator/internal/webhook/deploy	0.019s	coverage: 26.2% of statements
?   	github.com/kubeslice/worker-operator/pkg/events	[no test files]
?   	github.com/kubeslice/worker-operator/tests/emulator/hubclient	[no test files]
?   	github.com/kubeslice/worker-operator/tests/emulator/workerclient	[no test files]
?   	github.com/kubeslice/worker-operator/tests/emulator/workerclient/netop	[no test files]
?   	github.com/kubeslice/worker-operator/tests/emulator/workerclient/router	[no test files]
?   	github.com/kubeslice/worker-operator/tests/emulator/workerclient/sidecargw	[no test files]
ok  	github.com/kubeslice/worker-operator/tests/hub	14.230s	coverage: [no statements]
ok  	github.com/kubeslice/worker-operator/tests/spoke	53.971s	coverage: [no statements]
```

**Review by rahulsawra98 — APPROVED** [review]

**Pull request merged**